src/Nethermind/Nethermind.Optimism/Nethermind.Optimism.csproj

```diff
@@ -9,6 +9,7 @@
     <ProjectReference Include="..\Nethermind.Blockchain\Nethermind.Blockchain.csproj" />
     <ProjectReference Include="..\Nethermind.Core\Nethermind.Core.csproj" />
     <ProjectReference Include="..\Nethermind.Consensus\Nethermind.Consensus.csproj" />
+    <ProjectReference Include="..\Nethermind.Init\Nethermind.Init.csproj" />
```

Comment:
needed?

Reply:
yes, it was implicitly referenced by Nethermind.Merge.Plugin -> Aura -> Init.
So when i broke the coupling, it became needed